.circleci/config.yml

```diff
@@ -71,13 +78,17 @@ jobs:
                 then
                     CONTAINER_NAME="${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}"
             fi
-            # If not set, use testing repo name
-            if [ ! -n "{REPO_NAME}" ]
-              then
-              ${REPO_NAME}="https://www.github.com/${CONTAINER_NAME}"
-            fi
             echo "Container name is ${CONTAINER_NAME}"
-            echo "Repository name is ${REPO_NAME}"
+            # If not set, define REPO_NAME
+            if [ ! -n "${REPO_NAME:-}" ]
+                then
+                    LOCAL_REPO="${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}"
+                    REPO_NAME=/tmp/src
+                    echo "Repository name (REPO_NAME) is not defined, will build ${LOCAL_REPO} in ${REPO_NAME}"
+                    ls ${REPO_NAME}
+            else
+                echo "Repository name found defined for build: ${REPO_NAME}"
+            fi
             echo "2. Running jupyter-repo2docker..."
             echo "jupyter-repo2docker --debug --user-name username --user-id 1000 --no-run --image-name ${CONTAINER_NAME}:${DOCKER_TAG} ${REPO_NAME}"
             jupyter-repo2docker --debug --user-name username --user-id 1000 --no-run --image-name "${CONTAINER_NAME}:${DOCKER_TAG}" "${REPO_NAME}"
```

Comment:
`repo2docker --user-name username --user-id 1000 --no-run .` in the directory where the repository has been checked out should "just work". Or is `/tmp/src` that place already? Right now I am thinking someone had to copy the repo contents there.

Reply:
`/tmp/src` is just relevant for the CircleCI build (it's akin to the repo name, if you look at where it's defined):

https://github.com/vsoch/continuous-build/blob/cf477278c569815460fd2c49435a510098114011/.circleci/config.yml#L35

![image](https://user-images.githubusercontent.com/814322/43005626-503a594a-8be8-11e8-9ffe-33a47633dcf1.png)

This says the working directory for the repo is `/tmp/src`. I had to make this change because given that the user doesn't define an actual Github url, we want to build the local repository (the path/root to which is `/tmp/src`). It is just for circle, and doesn't have anything to do with the container. If the variable is not defined as a Github repo **or** folder, it errors out.